community/logging/src/main/java/org/neo4j/logging/FormattedLog.java

```diff
@@ -195,6 +197,8 @@ public FormattedLogger( Supplier<PrintWriter> writerSupplier, Object lock, Strin
         @Override
         protected void writeLog( PrintWriter out, String message )
         {
+            requireNonNull( message, "message" );
```

Comment:
This static method is new to me! Perhaps use it in `AbstractPrintWriterLogger` also? https://github.com/neo4j/neo4j/blob/2.3/community/logging/src/main/java/org/neo4j/logging/AbstractPrintWriterLogger.java#L51-L54